appsync-cognito-lambda-cdk/README.md

```diff
@@ -1,6 +1,6 @@
 # Authenticating an AppSync API using Amazon Cognito
 
-This CDK template deploys an AWS AppSync Api and a Cognito Userpool.
+This CDK template deploys an AWS AppSync API and a Cognito userpool.
```

Comment:
explain what the lambda function does. Why is needed?

Reply:
Taking a look now. It is a bit confusing. It looks like the original sample had a Cognito userpool created to authorize requests for the AppSync API, but also had queries and mutations that create and read from the userpool. 

I'll take a look at what the best way to showcase this is.

Reply:
updated